packages/api/src/webhooks/cannon.ts

```diff
@@ -31,7 +31,7 @@ export default class WebhookCannon {
   sendgridTemplateId: string;
   sendgridApiKey: string;
   supportAddr: [string, string];
-  taskScheduler: TaskScheduler;
+  taskScheduler: typeof TaskScheduler;
```

Comment:
This is a little weird/unnecessary. The `UpperCase` token refers to the type while `loweCase` is a variable/value. To fix this the right way, you should import the type instead of the singleton instance.

So the `import` line in the beginning should be
`import { TaskScheduler } from "../task/scheduler";`

and this line should stay as is, `TaskScheduler` being the type.

Edit: Actually, we should just remove the field from this `WebhookCannon` class and use the singleton directly. It was only a field before cause we needed to receive it in the constructor ✨